components/admin_console/admin_definition.jsx

```diff
@@ -521,6 +537,7 @@ const AdminDefinition = {
                 id: 'Groups',
                 component: GroupSettings,
             },
+            restrictedIndicator: getRestrictedIndicator(),
```

Comment:
How do we determine here if this feature is available in the current license or not?

Reply:
Tried it out and seems to be restricting as expected, just curious where do you check this so that it properly changes per license

Reply:
 `shouldDisplay` prop receives `license` and `subscription` to determine whether or not the restricted indicator should be shown for the feature item.
`value` prop determines the way it is shown, blocked (key) or available (high voltage)